views/files/files-grid.hbs

```diff
@@ -8,7 +8,9 @@
 							{{#ifneq this.securityCheck.status "blocked"}} fileviewer{{/ifneq}}"
 							{{#ifneq this.securityCheck.status "blocked"}} data-file-viewer-type="{{../this.type}}"{{/ifneq}}
 							{{#ifneq this.securityCheck.status "blocked"}} data-file-viewer-savename="{{../this.saveName}}"{{/ifneq}}
-							{{#ifneq this.securityCheck.status "blocked"}} data-file-viewer-id="{{../this._id}}"{{/ifneq}}>
+							{{#ifneq this.securityCheck.status "blocked"}} data-file-viewer-id="{{../this._id}}"{{/ifneq}}
```

Comment:
it is always better to use a button, if possible, instead of adding a role and tabindex, it's cleaner. Is there a reason why you didn't use a button here?

Reply:
This node contains other buttons, so it can't be changed to a button unfortunately.